DirectEmployers/MyJobs
# [PD-2712] chart type in payload

## Overview

This ticket involves including the chart type in the payload returned from the API. This is the default chart type of the grouping selected (i.e., if the data is grouped by Country, the chart type would be Map:World, etc)

In addition to this, this PR has expanded tests of the dynamic charting API. This wasn't really part of the ticket, but it just turned out that way. 

## Testing

* Navigate to https://secure.my.jobs/analytics/view/main
* Open browser inspector, head to the network tab.
* Reload the page and look for the call to "dynamic". The response should be a JSON object containing chart_type and group_by.

@mwarnsleyDE, I added a group_by on there.. It's technically already part of the response object under column_names[0]['key'], but I thought this may be more straight forward to use. If you don't need it, I can drop it.
